src/commands/client/login.js

```diff
@@ -33,11 +33,17 @@ module.exports = async args => {
     return error(`Failed to extract login creds from: ${token}`);
   }
 
-  const res = await invokeCommand('cli_login ' + token, token);
+  apiHost = apiHost
+    ? 'https://' + apiHost + '.nimbella.io'
```

Comment:
this should take the apihost as specified on the command line as is - on prem installations are not subdomains of `nimbella.io`. 

I think it's reasonable to check if `http[s]://` exists and prefix the host if not, but otherwise use the api host as given.